src/client/paths.rs

```diff
@@ -0,0 +1,17 @@
+use serde::Deserialize;
+
+#[derive(Clone, Debug, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Paths {
+    pub exchange_rates: String,
+    pub gold_prices: String,
+}
```

Comment:
I think it would be sufficient to store just string references `&str` in the Paths struct. Although the impact of current approach is negligible in case where there is (and typically would) a single NbpClient present at all times.